build.gradle

```diff
@@ -14,6 +14,21 @@ plugins {
     id 'com.google.dagger.hilt.android' version '2.44' apply false
     id 'io.github.gradle-nexus.publish-plugin' version '1.1.0'
     id 'org.jetbrains.kotlinx.kover' version '0.6.1'
+    id "org.jetbrains.dokka" version "1.9.0"
+}
+
+subprojects {
```

Comment:
Why do we need to do this subproject thing?

Reply:
We don't want to generate reference docs for our sample-app and I couldn't get the docs to build without defining the dokka configuration in the root-level `build.gradle`